packages/interface/src/components/complementary-area/style.scss

```diff
@@ -1,50 +1,30 @@
 .interface-complementary-area {
 	background: $white;
 	color: $gray-900;
-	overflow: visible;
 
 	@include break-small() {
-		z-index: auto;
-		height: 100%;
-		overflow: auto;
 		-webkit-overflow-scrolling: touch;
 	}
 
 	@include break-medium() {
 		width: $sidebar-width;
 	}
 
-	> .components-panel {
-		border-left: none;
-		border-right: none;
-		overflow: auto;
-		-webkit-overflow-scrolling: touch;
-		height: auto;
-		max-height: calc(100vh - #{ $admin-bar-height-big + $panel-header-height + $panel-header-height });
-		margin-top: -1px;
-		margin-bottom: -1px;
-		position: relative;
-
-		@include break-small() {
-			overflow: visible;
-			height: auto;
-			max-height: none;
-		}
+	.components-panel {
+		border: none;
 	}
 
-	> .components-panel .components-panel__header {
-		position: fixed;
-		z-index: z-index(".components-panel__header");
+	.components-panel__header {
+		position: sticky;
 		top: 0;
-		left: 0;
-		right: 0;
-		height: $panel-header-height;
+		z-index: z-index(".components-panel__header");
+
+		&.edit-post-sidebar__panel-tabs {
+			top: 48px;
 
-		@include break-small() {
-			position: inherit;
-			top: auto;
-			left: auto;
-			right: auto;
+			@include break-medium() {
```

Comment:
The small stuff appears to be working fine, so I ask only out of curiosity — why was this changed from small to medium?

Reply:
Its a different set of declarations. Its not that I changed a small to a medium, but more that I removed a small, and added a medium.